test/OpenTelemetry.Tests/Logs/LogRecordTest.cs

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 // </copyright>
 
-#if !NET452 && !NET46
+#if !NET461
```

Comment:
we'll get rid of netcore2.1 soon, so this'd be lot more simpler.! (2.1 is EOL next month!) We can start removing as soon as the current wave of .netframework removals.